datatracer/foreign_key/standard.py

```diff
@@ -60,11 +60,11 @@ def fit(self, list_of_databases):
         """
         X, y = [], []
         for metadata, tables in tqdm(list_of_databases, "extracting features"):
-            fks = metadata.get_foreign_keys()
-            fks = set([
-                (fk["table"], fk["field"], fk["ref_table"], fk["ref_field"])
-                for fk in fks
-            ])
+            fks = set()
+            for fk in metadata.get_foreign_keys():
+                if not isinstance(fk["field"], str):
+                    continue  # skip composite keys
+                fks.add((fk["table"], fk["field"], fk["ref_table"], fk["ref_field"]))
```

Comment:
Can we invert this logic?

Reading "If this is a string, add it", is much simpler than "if this is not a string, jump to the next one. If you did not skip the loop at the previous if, add it"

```
            fks = set()
            for fk in metadata.get_foreign_keys():
                if isinstance(fk["field"], str):
                    fks.add((fk["table"], fk["field"], fk["ref_table"], fk["ref_field"]))
```

Reply:
Sure.